ui_testing/testcases/basic_tests/test02_articles.py

```diff
@@ -464,3 +464,49 @@ def test019_download_article_sheet(self):
             fixed_sheet_row_data = self.fix_data_format(values)
             for item in fixed_row_data:
                 self.assertIn(item, fixed_sheet_row_data)
+
+    @parameterized.expand(['ok', 'cancel'])
+    def test020_create_approach_overview_button(self, ok):
+        """
+        Master data: Create: Overview button Approach: Make sure
+        after I press on the overview button, it redirects me to the active table
+
+        LIMS-6203
+        """
+        self.base_selenium.LOGGER.info('create new article.')
+        self.base_selenium.click(element='articles:new_article')
+        self.article_page.sleep_tiny()
+        # click on Overview, this will display an alert to the user
+        self.base_selenium.LOGGER.info('click on Overview')
+        self.base_selenium.click(element='articles:overview')
+        self.article_page.sleep_tiny()
+        # switch to the alert
+        if 'ok' == ok:
+            self.base_selenium.click(element='articles:confirm_overview')
+            self.article_page.sleep_tiny()
+            self.assertEqual(self.base_selenium.get_url(), 'https://automation.1lims.com/articles')
```

Comment:
Do not hard code the url here. here is how u need to do it
```suggestion
            self.assertEqual(self.base_selenium.get_url(), '{}articles'.format(self.base_selenium.url))
```